src/lib/__tests__/PageControls-test.js

```diff
@@ -3,6 +3,7 @@ import PageControls from '../PageControls';
 import Controls from '../Controls';
 import fullscreen from '../Fullscreen';
 import Browser from '../Browser';
+import { BROWSERS } from '../constants';
```

Comment:
Minor, but it looks like all these classes already import `Browser`. If we move the constant there, we could do something like the following to avoid the new dependency:

```
import Browser, { Browsers } from '.../../Browser';
```